config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
+  constraints CanAccessFlipperUI do
```

Comment:
I'm wondering if we can use Devise's [authenticate](https://rubydoc.info/github/heartcombo/devise/main/ActionDispatch%2FRouting%2FMapper:authenticate) method here

```ruby
authenticate(:admin_user) do
  mount Flipper::UI.app(Flipper) => '/flipper'
end
```

Also, would be possible to mount this as part of our admin panel? Maybe using `/admin/flipper` route

Reply:
I like the prefix `/admin` as well. Actually, the constraint could wrap the whole  `admin` namespace, except for the admin sign in/up routes